dojo/fixtures/test_type.json

```
@@ -47,5 +47,26 @@
         },
         "model": "dojo.test_type",
         "pk": 7
+    },
+    {
+        "fields": {
+            "name": "Wizcli IaC"
+        },
+        "model": "dojo.test_type",
+        "pk": 8
+    },
+    {
+        "fields": {
+            "name": "Wizcli Dir"
+        },
+        "model": "dojo.test_type",
+        "pk": 9
+    },
+    {
+        "fields": {
+            "name": "Wizcli Img"
+        },
+        "model": "dojo.test_type",
+        "pk": 10
```

This fixture is for test types that do not have a parser associated with them. The new test types you are adding here will be generated dynamically
```suggestion
    }
```

---

done @Maffooch